packages/test/src/test/responses.ts

```diff
@@ -21,18 +21,22 @@ export type MessageKind =
     | string
 
 export type ResponseMessage = {
-    kind   : MessageKind
+    kind: MessageKind
     message: string
-    data?  : Record<string, string>
+    data?: Record<string, string>
 }
 
 /**
  * Checks whether the _object_ is a ResponseMessage
  * @param object
  */
 export function isResponseMessage(object: { [key: string]: unknown }): object is ResponseMessage {
-    return object["kind"] !== undefined && typeof object["kind"] === "string" &&
-    object["message"] !== undefined && typeof object["message"] === "string"
+    return (
```

Comment:
Perhaps I would create two variables, one for `object["kind"]` and one for `object["message"]`, to make the comparison slightly more concise and obvious. But it is just my personal taste, please feel free to ignore this comment

Reply:
Agreed, done.